app/views/users/profile.html.erb

```diff
@@ -121,7 +121,13 @@
   <div style="text-align:center;" class="visible-sm visible-xs">
     <img class="img-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" src="<%= @profile_user.photo_path %>" />
   </div>
-  <%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @profile_user.id }", parent: :profile, user: @profile_user } %>
+
+
+    <p><a onClick="$('').toggle();" class="btn btn-default">User tags</a></p>
+	<div class="tags" style="display:none;">
+		<%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @user.id }", parent: :profile } %>
```

Comment:
```suggestion
		<%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @profile_user.id }", parent: :profile, user: @profile_user } %>
```

Reply:
Hey @richa031, you'll face this error when you try this PR. Also, after doing some changes, I think some more changes are required to complete. For now, do these changes, i will make some commits here. Thanks!